client/js/navigation/NavigationLayout.js

```diff
@@ -139,6 +139,20 @@ export default createDrawerNavigator(
   },
   {
     contentComponent: props => <DrawerContent {...props} />,
+    contentComponent: props => (
+      <View style={{ flex: 1 }}>
+        <SafeAreaView forceInset={{ top: "always", horizontal: "never" }}>
```

Comment:
Do you think your `SafeAreaView` should be the parent with `flex:1` and View component can be the child?

Reply:
I tried this but it throw the LOGOUT button to the top of the menu and all the other links disappeared.